src/emit/polkadot/storage.rs

```diff
@@ -66,9 +64,7 @@ impl StorageSlot for PolkadotTarget {
             scratch_len.into()
         );
 
-        log_return_code(binary, "seal_get_storage", exists);
-
-        let exists = binary.builder.build_int_compare(
+        let condition = binary.builder.build_int_compare(
```

Comment:
`exists` was supposed to mean "does the key exist". `condition` is not very clear